nss/build.rs

```diff
@@ -0,0 +1,6 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::configure()
+        .build_server(false)
+        .compile(&["../authd.proto"], &["../"])?;
```

Comment:
We can only run `cargo build` in a directory that contains a manifest file (i.e. Cargo.toml) which means we can only run it on the project root or on the crate directory, so we can rely on the relative path, as it is evaluated from the crate directory rather than where the command was called.